tests/lib/rules/i18n-no-collapsible-whitespace.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @fileoverview Disallow collapsible whitespace in translatable strings
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require( '../../../lib/rules/i18n-no-collapsible-whitespace' ),
+	config = { env: { es6: true } },  // support for string templates
+	formatMessage = require( '../../util/format-message' ),
+	RuleTester = require( 'eslint' ).RuleTester;
+
+//------------------------------------------------------------------------------
+// Constants
+//------------------------------------------------------------------------------
+
+const NO_NEWLINES = formatMessage( rule.ERROR_MESSAGE, { problem: ' (\\n)' } );
```

Comment:
Let's stick with var, at least for now.  We essentially document that that's what we're doing in `.eslintrc.json`.